cli/internal/plugins/plugins.go

```diff
@@ -143,45 +143,52 @@ func getBinarySuffix() string {
 	return ""
 }
 
-func (p *PluginManager) getDownloadPaths(org string, repo string, registry string, version string) downloadPaths {
-	pluginName := fmt.Sprintf("cq-source-%s_%s_%s", repo, runtime.GOOS, runtime.GOARCH)
-	destDir := filepath.Join(p.directory, "plugins", registry, org, repo, version)
+func (p *PluginManager) getDownloadPaths(org string, name string, registry string, version string) downloadPaths {
+	pluginName := fmt.Sprintf("cq-source-%s_%s_%s", name, runtime.GOOS, runtime.GOARCH)
+	destDir := filepath.Join(p.directory, "plugins", registry, org, name, version)
 	binarySuffix := getBinarySuffix()
 	destFile := filepath.Join(destDir, pluginName+binarySuffix)
 	zipPath := filepath.Join(destDir, pluginName+".zip")
-	binaryPathInZip := "plugins/source/" + repo + binarySuffix
-	downloadURL := fmt.Sprintf("https://github.com/cloudquery/cloudquery/releases/download/plugins/source/%s/%s/%s_%s_%s.zip", repo, version, repo, runtime.GOOS, runtime.GOARCH)
+	binaryPathInZip := "plugins/source/" + name + binarySuffix
+	downloadURLs := []string{
+		// new tags format e.g. https://github.com/cloudquery/cloudquery/releases/download/plugins-source-test-v1.1.5/test_linux_amd64
+		fmt.Sprintf("https://github.com/cloudquery/cloudquery/releases/download/plugins-source-%s-%s/%s_%s_%s.zip", name, version, name, runtime.GOOS, runtime.GOARCH),
+		// old tags format e.g. https://github.com/cloudquery/cloudquery/releases/download/plugins/source/test/v1.1.4/test_linux_amd64
+		fmt.Sprintf("https://github.com/cloudquery/cloudquery/releases/download/plugins/source/%s/%s/%s_%s_%s.zip", name, version, name, runtime.GOOS, runtime.GOARCH),
+	}
 	// we use convention over configuration and we use github as our registry. Similar to how terraform and homebrew work.
 	// For example:
 	// https://github.com/cloudquery/cloudquery/releases/download/plugins-source-test-v1.1.0/test_darwin_amd64.zip
 	if org != "cloudquery" {
 		// https://github.com/yevgenypats/cq-source-test/releases/download/v1.0.0/cq-source-test_linux_amd64.zip
-		downloadURL = fmt.Sprintf("https://github.com/%s/cq-source-%s/releases/download/%s/cq-source-%s_%s_%s.zip", org, repo, version, repo, runtime.GOOS, runtime.GOARCH)
-		binaryPathInZip = "cq-source-" + repo + binarySuffix
+		downloadURLs = []string{
+			fmt.Sprintf("https://github.com/%s/cq-source-%s/releases/download/%s/cq-source-%s_%s_%s.zip", org, name, version, name, runtime.GOOS, runtime.GOARCH),
```

Comment:
There's a bit of negative impact here as if the config points to a plugin with an old tag structure we print this log message twice